File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/UserStateUpdateUser.java
package com.dhsdevelopments.potato.clientapi.notifications;

import com.google.gson.annotations.SerializedName;

public class UserStateUpdateUser
{
    @SerializedName( "id" )
    public String id;

    @Override
    public String toString() {
        return "UserStateUpdateUser[" +
                       "id='" + id + '\'' +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/notifications/UserStateUpdateUser.kt
package com.dhsdevelopments.potato.clientapi.notifications

import com.google.gson.annotations.SerializedName

class UserStateUpdateUser {
    @SerializedName("id")
    lateinit var id: String

    override fun toString(): String {
        return "UserStateUpdateUser[id='$id']"
    }
}
